CONTRIBUTING.md

```diff
@@ -9,6 +9,18 @@ pay attention to a few things:
 4. your work must be signed (see below)
 5. you may contribute through GitHub Pull Requests
 
+# Coding and documentation Style
+
+- Code must be formated with `yarn run lint`
+- Code must pass `yarn run lint`
+- Code must pass `yarn run test`
```

Comment:
Does it make sense that code should pass both currently as one is an alias for the other ?